protocol/storage/fault_slasher.go

```diff
@@ -122,20 +122,26 @@ func (sfm *FaultSlasher) Slash(ctx context.Context, currentHeight *types.BlockHe
 			// Alternatives are discussed in #3358
 			continue
 		}
-		minerAddr, err := address.NewFromString(minerStr)
+
+		laterMinerActorAddr, err := address.NewFromString(lateMinerActor)
```

Comment:
out of curiosity why "later"?  It looks like it would make more sense to use "lateMinerActorStr" and then "lateMinerActor"?

Reply:
:open_hands: later because I have fat fingers, will correct it to `late`